resources/splash.ui

```diff
@@ -91,12 +91,31 @@ QWidget
     </rect>
    </property>
    <property name="text">
-    <string/>
+    <string>Downloading tk-framework-desktopserver v1.4.5 (10 of 15)</string>
```

Comment:
I presume you added this so you can preview what the message would look like in designer? I'm guessing it gets overwritten before it is displayed?